app/src/main/res/layout/label_editor.xml

```diff
@@ -8,26 +9,36 @@
     <FrameLayout
         android:id="@+id/frameLayout_labelEditor"
         android:layout_width="match_parent"
-        android:layout_height="match_parent">
-
-       <ViewSwitcher
-           android:id="@+id/viewSwitcher_visualizer_labelEditor"
-           android:layout_width="match_parent"
-           android:layout_height="400dp"
-           android:inAnimation="@android:anim/slide_in_left"
-           android:outAnimation="@android:anim/slide_out_right">
-           <VideoView
-               android:id="@+id/videoView_labelEditor"
-               android:layout_width="match_parent"
-               android:layout_height="match_parent"
-               android:layout_gravity="center" />
-
-           <TextureView
-               android:id="@+id/textureView_labelEditor"
-               android:layout_width="wrap_content"
-               android:layout_height="match_parent"
-               android:layout_gravity="center" />
-       </ViewSwitcher>
+        android:layout_height="400dp">
+
+        <ImageView
+            android:id="@+id/imageView_labelEditor"
+            android:layout_width="match_parent"
```

Comment:
Ids of View should be named by `[TYPE]_[USAGE]_[LAYOUT_FILE]`, so here it could be `imageView_stickmanBackground_labelEditor` by our own convention